README.md

```diff
@@ -97,31 +97,39 @@ spec:
     - type: "ImageChange"
   source:
     dockerfile: |
-      FROM <driver-toolkit image> as builder
+      ARG DTK
+      FROM ${DTK} as builder
+
+      ARG KVER
 
       WORKDIR /build/
 
       RUN git clone https://github.com/openshift-psap/simple-kmod.git
 
       WORKDIR /build/simple-kmod
 
-      RUN make all install # [KVER=<kenel version if not default>] [KMODVER=<kernel module version if not default>]
+      RUN make all install KVER=${KVER}
 
       FROM registry.redhat.io/ubi8/ubi-minimal
 
-      # better to copy it from builder rather than pulling it from the internet so it work also with disconnected envs
-      COPY --from=builder /usr/bin/kmod /usr/bin/
```

Comment:
Wasn't this change made for use with disconnected environments?

Reply:
Indeed. This is explained in the commit message:
```
we are using git clone anyway and installing kmod seems cleaner
```

Since we are using `git clone` in the Dockerfile, it won't work in disconnected envs anyway. In that case, there is no reason to manually "guess" what happens when we install `kmod` and copy the files but rather just install it directly on the last layer (giving that we are not gaining disconnected anyway).

Unless I am missing something...